app/helpers/symbol_helper.rb

```diff
@@ -0,0 +1,24 @@
+module SymbolHelper
+  def show_classes?(kind=params[:kind])
```

Comment:
Style/SpaceAroundEqualsInParameterDefault: Surrounding space missing in default value assignment.